components/keyboard_shortcuts/keyboard_shortcuts_modal/keyboard_shortcuts_modal.tsx

```diff
@@ -165,6 +193,31 @@ const KeyboardShortcutsModal = ({onExited}: Props): JSX.Element => {
                                 </div>
                             </div>
                         </div>
+
+                    </div>
+                    <div className='row'>
```

Comment:
I think we should show calls shortcuts only if the calls plugin is enabled (4/5),
cc// @esethna

Reply:
Interesting, what would be your biggest concern with this being displayed while the plugin is unavailable?

My view is that I'd consider this to be documentation which should be available to the user regardless of whether the feature is enabled or not but happy to hear other thoughts.

Reply:
To me, it’s just confusing if a feature isn’t available I don’t want extra information until we want to up-sell the feature, in that case, it would be great if we show locked Icon like we are doing with professional and enterprise level features
Like we are doing it for this calls button
<img width="177" alt="Screenshot 2022-09-01 at 12 41 24 PM" src="https://user-images.githubusercontent.com/16203333/187853974-527a8526-6e11-4ccc-bb26-7471008ee292.png">
but your point seems valid to me

Reply:
I understand your concern, just not sure myself on the best approach here. I'd only add that Calls is not just any other plugin and it's a technicality that it is one today but the plan is to make it an integrated part of the suite and not something that can be easily enabled/disabled.

Anyhow, I'll defer this decision to a PM, thanks for the good discussion :+1:

Reply:
Agree that in theory we should only be showing shortcuts if the functionality is available to users. However, if the near term plan is to make calls always enabled, then we can just take this as is to reduce complexity

Reply:
Fair enough. It's actually not too much complexity so I added a check. There is one bug though that will prevent the Calls shortcuts to show if the plugin is enabled after the user has loaded the page. I am looking to fix that but separately (see https://github.com/mattermost/mattermost-webapp/pull/10152 for context).

@AshishDhama Would you mind giving this another pass since I added this? Thanks.